pkg/acsengine/defaults-apiserver_test.go

```diff
@@ -188,8 +188,8 @@ func TestAPIServerConfigEnableRbac(t *testing.T) {
 	cs.Properties.OrchestratorProfile.KubernetesConfig.EnableRbac = pointerToBool(false)
 	setAPIServerConfig(cs)
 	a = cs.Properties.OrchestratorProfile.KubernetesConfig.APIServerConfig
-	if a["--authorization-mode"] != "Node" {
-		t.Fatalf("got unexpected '--authorization-mode' API server config value for EnableRbac=false: %s",
+	if _, ok := a["--authorization-mode"]; ok {
+		t.Fatalf("got unexpected '--authorization-mode' kubelet config value for EnableRbac=false: %s",
```

Comment:
Should be "got unexpected '--authorization-mode' API server config value for EnableRbac=false: %s"